nixos/modules/services/networking/ntp/chrony.nix

```diff
@@ -39,13 +41,45 @@ in
         '';
       };
 
+      package = mkOption {
```

Comment:
There is only one version of this package in nixpkgs and it has no variants. I don't know that this option provides any value over an overlay.

But then again... some people like that `package` option.

Reply:
Thank you for your reviews!

@aanderse Because chrony-4.0 is currently only available on unstable, having a `package` option offers a quick way to get it on a system that otherwise follows a stable channel (without having to wait for a backport, etc.). It’s certainly a workaround but might be appreciated by those interested in securing their time server connections ASAP:wink:

Reply:
Sounds good.

FYI you can always set `chrony = unstable.chrony;` in your overlay!